wrapper.ts

```diff
@@ -149,6 +152,17 @@ export class Pty {
       return;
     }
 
+    if (
+      size.cols < MIN_SIZE ||
+      size.cols > MAX_SIZE ||
+      size.rows < MIN_SIZE ||
+      size.rows > MAX_SIZE
+    ) {
+      throw new RangeError(
+        `Size (${size.rows}x${size.cols}) out of range: must be between ${MIN_SIZE} and ${MAX_SIZE}`,
+      );
+    }
+
```

Comment:
we should see where in the frontend it could possibly cause out of range too and prevent it there too (defense in depth!)